mixin: ruler emote eval failing alert (#3177)
grafana/mimir: docs/sources/operators-guide/mimir-runbooks/_index.md

- Increase the evaluation interval of the rule group. You can use the rate of missed evaluation to estimate how long the rule group evaluation actually takes.
- Try splitting up the rule group into multiple rule groups. Rule groups are evaluated in parallel, so the same rules may still fit in the same resolution.

### MimirRulerRemoteEvaluationFailing

> This alert fires when communication between `ruler` and `ruler-query-frontend` is failing to be established.

Reviewer: ```suggestion
This alert fires when communication between the `ruler` and the `ruler-query-frontend` fails to become established.
```

> The `ruler-query-frontend` component is exclusively used by the `ruler` to evaluate rule expressions when running in remote operational mode. If communication between these two components breaks, gaps are expected to appear in the case of recording rules or alerting rules will not fire when they should.

Reviewer: ```suggestion
The `ruler-query-frontend` component is exclusively used by the `ruler` to evaluate rule expressions while running in remote operational mode. If communication between these two components breaks, gaps appear, and recording rules or alerting rules do not fire when they should.
```
Please vet my changes for technical accuracy.

How to **investigate**:

> - Check the `Mimir / Remote ruler reads` dashboard to see if the issue is caused by failures or high latency
>   - **Failures**
>     - Check the `ruler-query-frontend` logs to find out more details about the error
>   - **High latency**
>     - Check the `Mimir / Remote ruler reads resources` dashboard to see if CPU or Memory usage increased unexpectedly

Reviewer: ```suggestion
- Check the `Mimir / Remote ruler reads` dashboard to see if the issue is caused by failures or high latency:
  - **Failures**
    - Check the `ruler-query-frontend` logs to find out more details about the error.
  - **High latency**
    - Check the `Mimir / Remote ruler reads resources` dashboard to see if CPU or memory usage increased unexpectedly.
```
